docs/gen-docs/kyma_deploy.md

```diff
@@ -28,7 +28,7 @@ kyma deploy [flags]
                                  	- Deploy a specific branch of the Kyma repository on kyma-project.org: "kyma deploy --source=<my-branch-name>"
                                  	- Deploy a commit (8 characters or more), for example: "kyma deploy --source=34edf09a"
                                  	- Deploy a pull request, for example "kyma deploy --source=PR-9486"
-                                 	- Deploy the local sources: "kyma deploy --source=local" (default "2.13.0")
+                                 	- Deploy the local sources: "kyma deploy --source=local" (default "2.13.1")
```

Comment:
I saw at least two recent commits from main branch is not in release-2.13 branch, is it intended?

Reply:
Not intended, will include them!